scripts/install-chef-suse.sh

```diff
@@ -1166,7 +1166,7 @@ touch $crowbar_install_dir/crowbar-installed-ok
 rm -f $crowbar_install_dir/crowbar_installing
 
 # activate provisioner repos
-curl -X POST http://localhost:3000/utils/repositories/activate_all.json
+crowbarctl repository activate-all
```

Comment:
Someone sanity-check me that this will be installed on non-mkcloud envs at this point?

Reply:
To be sure I would add it to Devel:Cloud:8/patterns-cloud.

Reply:
This turned out not to be necessary so this is ready to be re-reviewed